shlr/sdb/src/ht_uu.c

```diff
@@ -1,19 +1,112 @@
 /* sdb - MIT - Copyright 2018-2022 - ret2libc, pancake */
 
-#include "sdb/sdb.h"
+#include <stdint.h>
 #include "sdb/ht_uu.h"
-#include "ht.inc.c"
-
-// creates a default HtUU that has strings as keys
-SDB_API HtName_(Ht)* Ht_(new0)(void) {
-	HT_(Options) opt = {
-		.cmp = NULL,
-		.hashfn = NULL,
-		.dupkey = NULL,
-		.dupvalue = NULL,
-		.calcsizeK = NULL,
-		.calcsizeV = NULL,
-		.freefn = NULL
-	};
-	return Ht_(new_opt) (&opt);
+#include "sdb/heap.h"
+#include "../../../libr/include/cwisstable.h"  // XXX use cwisstable from sdb itself
```

Comment:
maybe better to move this into sdb and merge it and use it later when done :)

Reply:
also update the copyright header line, add your name and use 2023

Reply:
Yep. This is just something for me to iterate easily on the code from r2.